An additional link better than more reiterations: update README with ur_driver compatibility

jacknlliu opened the pull request:
add a **NOTE** about `ur_driver` compatibility and a few adjustments.

jacknlliu committed:
An additional link better than more reiterations: update REAME with ur_driver compatibility

gavanderhoorn commented:
Thanks for your PR.

While it is a good idea to draw attention to compatibility information, we don't want to have this information in multiple places. If anything changes we'll have to update it in multiple places as well, which adds to our maintenance burden.

Adding an additional link to the wiki pages is a good idea. If you could update your PR to add just the top bit, we could merge it.

jacknlliu committed:
update REAME with ur_driver compatibility

An additional link better than more reiterations: update REAME with ur_driver compatibility

jacknlliu committed:
Update README.md

jacknlliu commented:
Is it suitable from your side?

gavanderhoorn commented:
Yes. Thanks for taking the time to improve things, much appreciated! 👍
